### DISCUSSION TITLE
[Question][Hw2] Question about partial score

### DISCUSSION THREAD

**Discussion by uk-jeong** [prosyslab-classroom/cs424-program-reasoning#277] [Q&A]
Name: Lee JeongUk

I removed some of the default test cases from the test/dune file in order to obtain partial scores.
Would there be any issues with the final assignment grading if I remove the default test cases?

Thank you.

**Comment by goodtaeeun** [comment]
Hi, I am concerned that your default test set may not pass the coverage threshold.
Otherwise, it will be okay.
